lightning/src/routing/gossip.rs

```diff
@@ -128,6 +128,9 @@ pub struct NetworkGraph<L: Deref> where L::Target: Logger {
 	genesis_hash: BlockHash,
 	logger: L,
 	// Lock order: channels -> nodes
+	#[cfg(test)]
+	pub(crate) channels: RwLock<BTreeMap<u64, ChannelInfo>>,
```

Comment:
I made this public in tests in order to enable easy manipulation of during testing. Let me know if this is acceptable.